Scalar.Common/Maintenance/ConfigStep.cs

```diff
@@ -208,25 +209,86 @@ public bool TrySetConfig(out string error)
                 return false;
             }
 
-            return this.ConfigureWatchmanIntegration(out error);
+            this.GetConfigSettings();
+
+            if (this.existingConfigSettings == null)
+            {
+                return this.ConfigureWatchmanIntegration(out error);
+            }
+
+            GitProcess.ConfigResult config = null;
+            GitProcess.Result getResult = this.RunGitCommand(
+                process => {
+                    config = process.GetFromLocalConfig("feature.scalar");
+                    return null;
+                },
+                nameof(GitProcess.GetFromLocalConfig)
+            );
+            GitFeatureFlags flags = GitVersion.GetAvailableGitFeatures(this.Context.Tracer);
+            config.TryParseAsString(out string scalar, out error, defaultValue: "true");
+
+            if (scalar.Equals("false"))
```

Comment:
```suggestion
            if (StringComparer.OrdinalIgnoreCase.Equals(scalar, "false"))
```

You might also want to consider a `IsTruthy(string str)` utility method? We do this in GCM to be consistent with how Git itself parses config values like `1`, `TRUE`, or `yes`.

Reply:
We can consider that if we need to be robust in product-type config. For now, this config is intended only for a controlled group of dogfooders and we can ask them to use our preferred, exact config values.

I'll throw in the ignore-case stuff because that's easy enough.